spec/models/vacancy_spec.rb

```diff
@@ -586,7 +586,7 @@
       end
 
       it "returns an array with trust_uid and school_urns" do
-        expect(vacancy.organisation_urns.sort).to eq([{ trust_uid: "12345", school_urns: %w[100002 100003].sort }])
+        expect(vacancy.organisation_urns).to contain_exactly({ trust_uid: "12345", school_urns: %w[100002 100003].sort })
```

Comment:
This change needs to be reverted. 
I brought it to `main` in a different PR and we were still having failures based on the order of the `school_urns`.

Ended resolved with this commit: https://github.com/DFE-Digital/teaching-vacancies/commit/2240231c6f448c0d7bf249e8feebc2c82674bed6